lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -109,6 +112,11 @@ module Constants
     config_param :vm_id, :string, :default => nil
     config_param :vm_name, :string, :default => nil
 
+    # Set values from JSON payload with this key to the "trace" LogEntry field.
+    config_param :trace_key, :string, :default => DEFAULT_TRACE_KEY
+    # Whether to also keep the trace key/value in the payload.
+    config_param :keep_trace_key, :bool, :default => false
```

Comment:
Ah okay understood, I guess then my question is are there any situations where we'd want it both the Payload and the Entry? I'm trying to assess if this configuration is needed.